tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -1245,3 +1245,43 @@ def _phi(m):
         return np.sum(np.log(C)) / (N - m + 1.0)
 
     return np.abs(_phi(m) - _phi(m + 1))
+
+@set_property("fctype", "aggregate_with_parameters")
+def max_fixed_point(x, r, m):
```

Comment:
lets rename the feature calculator, what do you think about `max_langevin_fixed_point`